app/code/Scandiweb_Test/registration.php

```diff
@@ -0,0 +1,6 @@
+<?php 
+	\Magento\Framework\Component\ComponentRegistrar::register(
```

Comment:
Please import the `\Magento\Framework\Component\ComponentRegistrar` class

Reply:
You can import the class like `use Magento\Framework\Component\ComponentRegistrar;` 
And then you can use `ComponentRegistrar::register` and `ComponentRegistrar::MODULE` directly